.gitignore

```diff
@@ -189,13 +189,13 @@ src/include/pbs_version.h
 src/include/pbs_config.h
 src/include/pbs_config.h.in
 src/include/pbs_config.h.in~
-src/server/queue_attr_def.c
-src/server/resc_def_all.c
-src/server/resv_attr_def.c
-src/server/sched_attr_def.c
-src/server/svr_attr_def.c
-src/server/job_attr_def.c
-src/server/node_attr_def.c
+src/lib/Libattr/queue_attr_def.c
+src/lib/Libattr/resc_def_all.c
+src/lib/Libattr/resv_attr_def.c
+src/lib/Libattr/sched_attr_def.c
+src/lib/Libattr/svr_attr_def.c
+src/lib/Libattr/job_attr_def.c
+src/lib/Libattr/node_attr_def.c
```

Comment:
Why the move to Libattr?  I don't see any particular reason in the refactoring code.  I don't necessarily think it's a  bad place for it though.